itest/server_harness.go

```diff
@@ -0,0 +1,106 @@
+package itest
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net"
+	"path"
+	"sync"
+	"time"
+
+	"github.com/lightninglabs/loop/looprpc"
+	"github.com/lightninglabs/pool/auctioneerrpc"
+	"github.com/lightningnetwork/lnd/cert"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+const (
+	// DefaultAutogenValidity is the default validity of a self-signed
+	// certificate. The value corresponds to 14 months
+	// (14 months * 30 days * 24 hours).
+	DefaultAutogenValidity = 14 * 30 * 24 * time.Hour
+)
+
+type loopPoolServer struct {
+	auctioneerrpc.UnimplementedChannelAuctioneerServer
```

Comment:
Ohh, very cool hack here to not have to manually declare everything needed to pose as a loop/pool server!